examples/flutter_gallery/test_driver/transitions_perf_test.dart

```diff
@@ -46,6 +50,41 @@ const List<String> demoNames = const <String>[
   'Typography'
 ];
 
+
+Future<Null> saveDurationsHistogram(List<Map<String, dynamic>> events) async {
+  final Map<String, List<int>> durations = new Map<String, List<int>>();
```

Comment:
OMG, this looks worse than Java. Not your fault. We really need to relax our style guide for obvious cases.